charts/rd-judicial-data-load/values.yaml

```diff
@@ -14,6 +14,8 @@ job:
     JRD_MQ_TOPIC_NAME: rd-judicial-topic-{{ .Values.global.environment }}
     JRD_MQ_TRUST_ALL_CERTS: false
     JRD_DATA_PER_MESSAGE: 500
+    IDAM_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
+    OAUTH2_REDIRECT_URI: https://rd-professional-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/oauth2redirect
```

Comment:
I think this should not be professional-api

Reply:
:)